interfaces/builtin/dsp.go

```diff
@@ -49,10 +49,16 @@ const ambarellaDspConnectedPlugApparmor = `
 # processor
 /dev/ucode rw,
 
-# The iav device node is the device node exposed for the specific IAV linux
-# device driver used on the ambarella device
+# The iav device node is the device node exposed for the specific IAV driver
+# that includes a set of Linux device drivers that encapsulate the lower-level
+# complexities of the CV2x / S6Lm DSP core functionalities
 /dev/iav rw,
 
+# The cavalry device node is the driver for the CV2x vector processor (VP)
+# that includes a set of Linux device drivers that encapsulate the lower-level
```

Comment:
what "includes a set of device drivers" mean ?   are there multiple drivers all exposed through /dev/cavalry?